src/app/components/FacetSidebar/FacetSidebar.jsx

```diff
@@ -179,11 +179,13 @@ class FacetSidebar extends React.Component {
                 return (
                   <label
                     key={j}
+                    id={`${facet.field}-${valueLabel}`}
                     htmlFor={`${facet.field}-${valueLabel}`}
                     className={`nypl-bar_${percentage}`}
                   >
                     <input
                       id={`${facet.field}-${valueLabel}`}
+                      aria-labelledby={`${facet.field} ${valueLabel}`}
```

Comment:
i think 0.1.10 uses the latest markup for the facet:

````html
<label id="my-facet-label-id" for="my-facet-value-id" class="nypl-bar_[1 to 100]">
  <input aria-labelledby="my-facet-label-id my-facet-value-id" id="my-facet-value-id" type="checkbox" name="my-name" value="my-value" />
  <span class="nypl-facet-count">[estimated result count]</span>
  The facet text
</label>
````

basic structure is: `label>input+span.nypl-facet-count+facet-value`

(only 1 span throughout so i think that fixes a few of the issues you've been having)